### DISCUSSION TITLE
How do I add Subject metadata to my NWB file using R?

### DISCUSSION THREAD

**Discussion by nrsc** [NeurodataWithoutBorders/helpdesk#87] [General]
Hello all. While preparing a dandiset for publication I had to update some metadata in the subject section of the nwb files. I was able to write a script that takes advantage of the rhdf5 package (https://bioconductor.org/packages/release/bioc/html/rhdf5.html) to write subject_id, sex, age, and species into the nwb file. The script can be found at https://github.com/nrsc/nwbR/blob/main/R/write_exp_metadataNWB.R. Plan is to develop this repository into an R package hosted on CRAN. Currently the repo is in the initial stages of development. Feel free to reach out about any use cases. Cheers

**Comment by rly** [comment]
Thanks for sharing @nrsc !
